src/server/plugins/engine/plugin.ts

```diff
@@ -80,9 +80,12 @@ export const plugin = {
     const forms = server.app.forms
 
     configs.forEach((config) => {
```

Comment:
This caught me out a bit, having a clash with `config` (import) versus `config` scoped

Should we rename it?

```suggestion
    configs.forEach((form) => {
```

Reply:
Yeah this caught me out too. I had previously renamed the local forEach variable in the earlier PR.

Reply:
Resolved https://github.com/DEFRA/forms-runner/pull/57/commits/96c9c1fff40755b20a0201e94032f52841ad43d6